### PR TITLE
chore: deploy connect ui

### PR DESCRIPTION
**Pull request by bodinsamuel**
## Describe your changes

- Allow manual deployment of Connect UI

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -118,3 +118,21 @@ jobs:
                   echo "Deploying $IMAGE to $SERVICE_ID"
 
                   curl -sS --fail-with-body --request POST "https://api.render.com/v1/services/$SERVICE_ID/deploys" --header "authorization: Bearer ${{ secrets.RENDER_API_KEY }}" --data "{ \"imageUrl\": \"$IMAGE\" }"
+
+    deploy_connect_ui:
+        if: inputs.service == 'connect_ui'
```
Comment (TBonnin) [inline .github/workflows/deploy.yaml:124]: you need to add `connect_ui` to the list of `input.service` at the top of this file?

File: .github/workflows/deploy.yaml [hunk 2]
```diff
@@ -118,3 +118,21 @@ jobs:
...
+              with:
+                  username: ${{ secrets.DOCKER_USERNAME }}
+                  password: ${{ secrets.DOCKER_PASSWORD }}
+            - name: Deploy connect
+              run: |
+                  SERVICE_ID=${{ fromJson('{ "production": "srv-crtut51u0jms73bts8fg", "staging": "srv-cmj6t6da73kc739ol660" }')[inputs.stage] }}
+                  COMMIT="${{ github.sha }}"
+                  echo "Deploying $COMMIT to $SERVICE_ID"
+
+                  curl -sS --fail-with-body --request POST "https://api.render.com/v1/services/$SERVICE_ID/deploys" --header "authorization: Bearer ${{ secrets.RENDER_API_KEY }}" --data '{ "commitId": "${COMMIT}" }'
```
Comment (TBonnin) [inline .github/workflows/deploy.yaml:139]: do we want to clear the cache?

**Review by TBonnin — APPROVED** [review]
you forgot to add connect_ui to the list of input.services but lgtm otherwise

File: .github/workflows/deploy.yaml (hunk 2 above)
Comment (bodinsamuel) [reply .github/workflows/deploy.yaml:139]: nope, it's not_clear by default

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (bodinsamuel) [reply .github/workflows/deploy.yaml:124]: 🤦🏻

**Pull request merged**